src/transformStateWithClones.js

```diff
@@ -7,7 +7,36 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateCopy = { ...state };
+  const changesArray = [];
+
+  for (const deed of actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```
I think it looks better